adding an action step to enforcement guide (#69)
jupyter/governance: conduct/code_of_conduct.md

This code applies equally to founders, developers, mentors and new community
members, in all spaces managed by Project Jupyter (including IPython). This
includes the mailing lists, our GitHub organizations, our chat rooms, in-person

> events, community forum, and any other forums created by the project team. In addition,

Reviewer: Worth being explicit about it being "the Discourse community forum"? Otherwise the language "c. forum and other forums" reads a bit funny. We only have Discourse and so I don't see any danger in being explicit about it. Very minor nit.

Author: done - added in a link to the forum as well

   automatically imply an admission of guilt.

## Responding to inappropriate behavior

In some cases, individuals may violate the CoC in online or in-person situations.
If any community member deems the action to be harmful to the community,
they may act immediately to defuse the situation.
This includes actions directed at the situation, rather than at an individual, such
as locking a thread, *temporarily* suspending a user's account in the
community forum, calling a meeting break, or generally acting to
de-escalate the situation. Follow-up your actions by

> [reporting the incident to the CoC committee](#Reporting).

Author: @rgbkrk what do you think about this wording? better?

Reviewer 2: Much better, thank you.

If the incident involves physical danger, or involves a threat to anyone's safety

> (e.g. threats of violence), any member of the community may -- and should -- act

Reviewer 3: ```suggestion
(such as threats of violence), any member of the community may -- and should -- act
```

> (e.g. threats of violence), any member of the community may -- and should -- act

Reviewer 3: ```suggestion
(threats of violence, for example), any member of the community may -- and should -- act
```

   automatically imply an admission of guilt.

## Responding to inappropriate behavior

In some cases, individuals may violate the CoC in online or in-person situations.
If any community member deems the action to be harmful to the community,
they may act immediately to defuse the situation.
This includes actions directed at the situation, rather than at an individual, such
as locking a thread, *temporarily* suspending a user's account in the

> community forum, calling a meeting break, or generally engaging in actions

Reviewer 4: ```suggestion
community forum, calling a meeting break, or generally acting to
```

> aimed at de-escalating the situation.

Reviewer 4: ```suggestion
de-escalate the situation.
```

> If the action is ongoing or requires a response directed at a specific individual,

Reviewer 2: For several of the actions above, they should _definitely_ be reported rather than allowed to resurface as ongoing.

Author: Good point - I don't want to suggest that violations should *generally* need to be ongoing before reporting.

Are there a subset of actions that require formal CoC committee review? Or is it the case that *any* violation of the CoC also requires a review? (this is a question about what the current process is, rather than how it should be - I just don't know the answer).

Author: actually reading back through my original comment on this PR (sorry it's been a few months so I haven't thought about this in a while), I think any "action" taken in response to a CoC violation should be reported, so perhaps the language about "ongoing" should just be removed?

Here I am thinking from the perspective of a moderator in the community forum - if I lock a thread because somebody is being a jerk, should I report that to the CoC each time? And would that be different from reporting CoC violation that requires a formal investigation etc (or perhaps that is purely for the CoC committee to decide after the report?)

Reviewer 2: Yeah I think removing "ongoing" is good.

then the incident should be [reported to the CoC committee](#Reporting).

If the incident involves physical danger, or involves a threat to anyone's safety
(e.g. threats of violence), any member of the community may -- and should -- act
unilaterally to protect the safety of any community member.
This can include contacting law enforcement, other members of the Jupyter community,
or other local personnel that are present.

In situations where an individual community member acts unilaterally,
they must inform the Code of Conduct committee as soon as possible,

> and report their actions for review by e-mailing

Reviewer 4: ```suggestion
by e-mailing
```
